### ISSUE TITLE
Replace akka-paradox plugin with paradox

### ISSUE THREAD

**Issue by softinio** [apache/incubator-pekko#32]
As a quick way to remove the akka branding, lets replace [akka-paradox](https://github.com/akka/akka-paradox) plugin with [paradox](https://github.com/lightbend/paradox) basic theme. We can make better with more Pekko branding later.

As discussed in https://github.com/apache/incubator-pekko/issues/28 with @nvollmar

**Comment by softinio** [comment]
If no objection, I will start working on this tomorrow.

**Comment by mdedetrich** [comment]
@seglo @softinio @pjfanning So I tried to do just simply replace the akka paradox theme with the standard one and this made me fall down a rabbit hole. Here are the problems

* Due to the current Pekko sbt build only loading on an ancient JDK 1.8 version (it actually refuses to load otherwise) we are forced to downgrade to a version of parboiled that is bytecode compatible with JDK 1.8. I have managed to do this locally
* Although initially it might make sense to just replace `addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.44")` with the various sbt paradox plugins initially, this will have to be repeated for every single pekko project. Furthermore at some point we would actually like to create a pekko theme, and in order to not repeat boilerplate in the 10+ repos we maintain it makes sense to create a `sbt-paradox-pekko` sbt plugin akin to [sbt-paradox-akka](https://github.com/akka/akka-paradox). Initially it can be blank and essentially just contain the following dependencies
  ```sbt
  addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % 0.10.3")
  addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "1.0.0")
  addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-project-info" % "3.0.0")
  ```
  but later on an actual css/js theme can be added. I already have a sample `sbt-paradox-pekko` locally .
